rfcs/text/0033-tty-output.md

```diff
@@ -62,7 +67,7 @@ Stage 1: Provide a high-level description of example sources of data. This does
   {
     event: {
       kind: 'event',
-      action: 'text_output' (for now the only action type, though once could imagine values like: text_input, binary_output, binary_input?)
+      action: 'text_output' (for now the only action type, though one could imagine values like: text_input, binary_output, binary_input?)
```

Comment:
no question mark nec on binary_input (?) - e.g. in the future it could track binary data read from a file.